iothub/device/src/Twin/TwinProperties.cs

```diff
@@ -23,11 +25,23 @@ protected internal TwinProperties(DesiredProperties requestsFromService, Reporte
         /// <summary>
         /// The collection of desired property update requests received from service.
         /// </summary>
+        [JsonProperty("desiredProperties")]
         public DesiredProperties Desired { get; }
 
         /// <summary>
         /// The collection of twin properties reported by the client.
         /// </summary>
+        [JsonProperty("reportedProperties")]
         public ReportedProperties Reported { get; }
+
+        /// <summary>
+        /// Gets the Twin as a JSON string
+        /// </summary>
+        /// <param name="formatting">Optional. Formatting for the output JSON string.</param>
+        /// <returns>JSON string</returns>
+        public string ToJson(Formatting formatting = Formatting.None)
```

Comment:
I'm sure you've tested this locally, but can you also add a test for this verifying that the payload is serialized as expected.